pkg/virt-handler/notify-server/server.go

```diff
@@ -154,11 +155,31 @@ func RunServer(virtShareDir string, stopChan chan struct{}, c chan watch.Event,
 	case <-done:
 		log.Log.Info("notify server done")
 	case <-stopChan:
-		log.Log.Info("stopping notify server")
-		grpcServer.Stop()
+		grpcServerStop(grpcServer)
 		sock.Close()
 		os.Remove(sockFile)
 	}
 
 	return nil
 }
+
+func grpcServerStop(grpcServer *grpc.Server) {
+	gracefulShutdownTimeout := 10
```

Comment:
Isn't it better to parametized this parameter and not use hard coded value (10)?

Reply:
I don't think that it makes much of a difference if it is only used once in the project and only with this value. It can always be made a parameter if needed.

Reply:
This means that graceful shutdown will always be set to 10 minutes hardcoded while sometimes you as a user would like to reduce or increase the time, no?

Reply:
10 seconds, and i am working on #3730 where the client ERRORs wont even cause the GRPC server restart.
btw if an action item is needed, it would be better to do it first on master and then on the release i guess ?
thank

Reply:
right, 10 seconds (fixed the comment). it's always good to have a back pressure ability like grace full shutdown when needed

Reply:
What are the action items please? 
 - Add a TO param to RunServer, that will affect the timeout ?
Just a note, when all is fine its matter of less than a second to stop the server,
Didnt try it with 100 VMs but.

Can it be done in a different PR please?
(this one is a cherry-pick from master, so better it would be the same i think)

Reply:
@oshoval I think that @oshoval just wants to understand the context. I don't think any action is needed.